lib/awful/mouse/init.lua

```diff
@@ -297,8 +297,8 @@ function mouse.object.set_current_client() end
 
 function mouse.object.get_current_wibox()
     local obj = capi.mouse.object_under_pointer()
-    if type(obj) == "drawin" then
-        return obj
+    if type(obj) == "drawin" and obj.wibox then
+        return obj.wibox()
```

Comment:
That will be problematic, it is that way to avoid a leak in Lua 5.1. The only way I found to fix it is by using an upvalue weak table. I will rename this to `get_wibox()`. Otherwise, it would look like `.wibox[1]` and that's way worst than `wibox()`